app/Services/UserService.php

```diff
@@ -115,7 +115,7 @@ public function createUser(array $attrs, array $roles = []): User
         $this->calculatePilotRank($user);
         $user->refresh();
 
-        event(new UserRegistered($user));
+        event(new Registered($user));
```

Comment:
Is this a Laravel built-in event now?

Reply:
Yeah this is the build-in event to send the email verification link

https://laravel.com/docs/10.x/verification#model-preparation